python/heterocl/tvm/build_module.py

```diff
@@ -380,6 +380,9 @@ def lower(sch,
     stmt = ir_pass.AdjustBufferBinding(stmt, arg_list)
     stmt = ir_pass.InferStream(stmt, arg_list)
     stmt = ir_pass.AdjustBufferBinding(stmt, arg_list)
+    # perform layout transformation
+    stmt = ir_pass.TransformLayout(stmt, arg_list)
+    stmt = ir_pass.AdjustBufferBinding(stmt, arg_list)

```

Comment:
What does AdjustBufferBinding do? Why is it called multiple times after each pass?